### DISCUSSION TITLE
Steven Fabre - LiveBlocks

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#165] [Announcements]
## Background

- https://liveblocks.io/
- [@stevenfabre](https://twitter.com/stevenfabre)
- https://stevenfabre.com/
- https://gifmock.com/

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today is Steven Fabre, CEO and co-founder of liveblocks. Steven, thanks for joining us. I'm really excited to talk about liveblocks, but before that would you like to tell our listeners a bit more about yourself?

## Topics

- liveblocks
- transitioning from a product company (invision) to a platform company (liveblocks)
- challenges of realtime collaboration

### Background 

- [x] Before liveblocks was the product you and Guillaume were building a google slides competitor. What convinced you that the real time collaboration aspect was were the real product opportunity is?

### Live Blocks

- [x] What is liveblocks and why would someone want to us it?
- [x] How does liveblocks differ from other collaboration backend services?
- [x] How does it deal with conflicts?
- [x] How does it deal with offline?
- [x] Did you experience creating a design tooling startup influence how the company approached DX for liveblocks?
- [x] What types of collaborative presence can I add to my app?
- [x] What devtools does liveblocks have?
- [x] Can you an monitor apps?
- [x] The website teases some new features that seem to go outside of just the primitives for collaborative experiences (comments, access). How will these features integrate with my backend? Does liveblocks become a user management system as well?
- [x] With pricing based on monthly active users, what happens if a page goes viral for a few days?
- [x] What are some of your favorite things built on liveblocks? Anything unexpected?

### Future

- [ ] ~What other types of collaborative experiences do you want to enable in the future?~
- [x] What do you think the future of apps is? More collaboration?

**Comment by hipstersmoothie** [comment]
Tooltips:

- https://github.com/emilkowalski/sonner
- https://github.com/vadimdemedes/ink-ui

**Comment by stevenfabre** [comment]
https://www.plasmo.com/

**Comment by just-be-dev** [comment]
- https://github.com/microsoft/guidance
- https://beepberry.sqfmi.com/
